src/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="./css/stylesheet.css">
+  <title>Library App</title>
+</head>
+<body>
+  <div class="button-container">
+    <button id="newBook" onclick="newBook()">New Book</button>
+    <button onclick="render()">Show books</button>
```

Comment:
I don't see need for this `button` since the books display already.

Reply:
removed